devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -203,7 +181,7 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
-    if (cmd.value == 0 && device.latestState("color").value != "#ffffff") {
+    if (device.latestState("color") == null || (cmd.value == 0 && device.latestState("color").value != "#ffffff")) {
         sendEvent(name: "color", value: "#ffffff", displayed: true)
```

Comment:
I added a comment just above the code. I am going to double check querying the device for its current color, but if I recall we couldn't query for all color components at once and had to query red, green, and blue separately.